apps/yggdrasil/priv/repo/migrations/20160229163000_create_role_resource.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Yggdrasil.Repo.Migrations.CreateRoleResource do
+  use Ecto.Migration
+
+  def change do
+    create table(:roles_resources) do
+      add :role_id, references(:roles, on_delete: :nothing)
+      add :resource_id, references(:resources, on_delete: :nothing)
+      add :permission_id, references(:permissions, on_delete: :nothing)
+
+      timestamps
+    end
+    create index(:roles_resources, [:role_id])
+    create index(:roles_resources, [:resource_id])
+    create index(:roles_resources, [:permission_id])
```

Comment:
Forgot I had eliminated a table here, sorry shuffled around so much stuff in the last day or so.

That makes sense, because if I eliminate the id column on the table role_id for example would not be the "primary key".

This table should not be huge over it's lifetime or inserted into to frequently, so I'm not to concerned with insertions, and will largely be read more frequently I would think.